src/NotesApplication/README.txt

```diff
@@ -0,0 +1,19 @@
+
+
+schema:
+
+    NoSQL:
+
+    Database: "NotesAplication"
+        - Collection: "Notes"
+            - Database{
+                documentKey: <string>
+                locationOfBlob: HashTable <key: int, value: location>
```

Comment:
Honestly, lets just try out and see if it saves ordering in the database. Lets turn this into an array